src/test/java/pages/SearchPage.java

```diff
@@ -54,8 +55,24 @@ public void clickSaveAsButton(){
 
     public void clickNewFilterButton() { $(By.cssSelector("a.new-search.aui-button.aui-button-light")).click(); }
 
-    public boolean atRequiredPage() //проверяет, что мы на searchPage.
-    { Assert.assertEquals(title(), "Issue Navigator - Hillel IT School JIRA");
-        return true;
-    }
+    public void clickButtonChangeViews(){
+        $(".aui-buttons").click();}
+
+    public void clickDetailView(){
+        $("a.aui-list-item-link[data-layout-key='split-view'").click(); }
+
+    public void clickListView(){
+        $("a.aui-list-item-link[data-layout-key='list-view']").click(); }
+
+    public void titleTestNewIssue(){
+        $(By.cssSelector("[title='[Test Automation] Test New Issue']")).isDisplayed(); }
```

Comment:
Maybe it's better to pass "[Test Automation] Test New Issue" string as an argument ?
public void .....(String issueName){
      $(By.cssSelector("[title=" + issueName + "]")).isDisplayed(); 
}

Reply:
@vadlvadl 
I do not agree, because this method checks the avaibilyty of issue in results, wish we enter in jql request